### ISSUE TITLE
Add the ability to export modpacks in the Twitch export format?

### ISSUE THREAD

**Issue by Valiec** [deltamc-project/deltamc-python#51]
For compatibility with Curse modpacks, maybe make it possible to import/export Twitch exports, since there are a lot of mod packs using these already? I don't mean getting modpacks from Curse (at least for now), I just mean that if you have a Twitch export file (either from Curse or from the Twitch launcher, you should be able to import it.

I wouldn't remove the current .modpack file feature though (at least until I'm sure it has no benefits over Twitch exports).

**Comment by williambl** [comment]
Good idea. I'll have a look into the twitch modpack format (is it the same as multimc? It might be, seeing as twitch modpacks can be imported into it...)
